pyvrp/diversity/tests/test_broken_pairs_distance.py

```diff
@@ -8,10 +8,10 @@
 def test_broken_pairs_distance():
     data = read("data/OkSmall.txt")
 
-    sol1 = Solution(data, [[1, 2, 3, 4], [], []])
+    sol1 = Solution(data, [[1, 2, 3, 4]])
     sol2 = Solution(data, [[1, 2], [3], [4]])
-    sol3 = Solution(data, [[3], [4, 1, 2], []])
-    sol4 = Solution(data, [[4, 3, 2, 1], [], []])
+    sol3 = Solution(data, [[3], [4, 1, 2]])
+    sol4 = Solution(data, [[4, 3, 2, 1]])
```

Comment:
Since I removed support for empty routes, for most tests they were already removed but I actually forgot to remove these with the previous PR (removing empty routes from Solution)